packages/udes-ui-validation/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "udes-ui-validation",
+  "version": "0.0.1",
+  "description": "Package to validate react component fields.",
+  "main": "webpack.config.js",
+  "scripts": {
+    "jest:lerna": "jest --coverage",
+    "test": "lerna run jest:lerna",
+    "lint": "eslint src -c .eslintrc.json --ext js,jsx,ts,tsx; exit 0",
+    "check": "lint-staged"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/unipartdigital/udes-ui.git"
+  },
+  "keywords": [
+    "udes",
+    "udes-ui",
+    "udes-ui-validator"
+  ],
+  "author": "Robert Smith <robert.smith@unipart.io>",
+  "license": "GPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/unipartdigital/udes-ui/issues"
+  },
+  "homepage": "https://github.com/unipartdigital/udes-ui#readme",
+  "dependencies": {
```

Comment:
See @udes-ui/core for how to layout dependancies so we don't run in to versioning issues when installing the package

Reply:
i went through and removed the ^'s in my update to it, i'll make sure they are all gone

Reply:
You need to see how `peerDependencies` and `devDependencies` are used